### ISSUE TITLE
Remaining problem in unisusie pipeline

### ISSUE THREAD

**Issue by SamuelSun2000** [cumc/xqtl-pipeline#527]
@hsun3163 Please note that `uni_susie_2` in https://github.com/cumc/xqtl-pipeline/blob/main/code/fine_mapping/SuSiE/SuSiE.ipynb cannot be successfully executed yet. There are two problems:

1. Some of the rds files will be blank, and cannot be handled by readRDS() function.  Adding `improt os; skip_if(os.path.getsize(_input) == 0)` may help.
2. Because now output rds is only one susie_object, there will be errors about the variable `susie_list` in `uni_susie_2`. We no longer can use `susie_list[[i]]$pip` to extract pip. Instead, we should directly use `susie_list$pip`.

Because I currently don't know the strategy for correcting this step -- whether I should make a vcf for per gene per tissue, or integrate the result per gene in all tissues.  So I leave this issue here as a reminder.

**Comment by gaow** [comment]
@SamuelSun2000 could you provide me a minimal working example for this? (remember the style of the RNA-seq protocol you have tried last year we have the MWE). I'm swamped today until late in the PM .. but this one is now on the top of my TODO to finalize so others can use.

**Comment by gaow** [comment]
I see. @SamuelSun2000 the MWE you provided me can they reproduce these issues?

**Comment by SamuelSun2000** [comment]
Yes, both of them.

**Comment by hsun3163** [comment]
I was thinking, perhaps we should take away the uni_susie_2/3 altogether. And make the transform to vcf step another utility in the susie_post_proc notebook. Along with the code that produces the three summary tables.

**Comment by gaow** [comment]
yes i remember your proposal. let me take a look maybe this evening or tmr
